backend/src/test/java/gov/cdc/usds/simplereport/service/TestOrderServiceTest.java

```diff
@@ -189,12 +186,9 @@ void roundTrip() {
     verify(patientLinkService).createPatientLink(any());
 
     // make sure the corrected event is sent to storage queue
-    verify(testEventReportingService).report(testEventArgumentCaptor.capture());
     verify(fhirQueueReportingService).report(any());
-    TestEvent sentEvent = testEventArgumentCaptor.getValue();
-    TestResult testResult = sentEvent.getCovidTestResult().get();
-    assertThat(sentEvent.getPatient().getInternalId()).isEqualTo(patient.getInternalId());
-    assertThat(testResult).isEqualTo(TestResult.POSITIVE);
+    // improve fhir verification? test was only asserting csv / covid pipeline values
+    // was asserting: patient ids match, test results match
```

Comment:
should this be a ticket? I kinda feel like this comment is just going to sit here otherwise